helm/charts/determined/templates/master-config.yaml

```diff
@@ -35,7 +35,13 @@ data:
       port: {{ .Values.db.port }}
       name: {{ .Values.db.name | quote }}
 
-    http_port: {{ .Values.httpPort }}
+    {{- if .Values.tlsSecret }}
+    security:
+      tls:
+        cert: {{- include "determined.secretPath" . | indent 1 -}}tls.crt
```

Comment:
non-blocking: Would be much more natural to just drop the `-`s and not use `indent`, here and in various other places.

Reply:
Oh, I missed your other comment -- does this run into the same inline issue you mentioned? Don't worry about this.

Reply:
I got this one to work, key was removing the leading `-` also.